test/highlight/expressions.gleam

```diff
@@ -6,3 +6,7 @@ case "12345" {
   _ -> ""
   //   ^ string
 }
+
+-x
+^ operator
+ ^ variable
```

Comment:
I believe these need to be under comments to be recognized by the highlighting tests. We may need to tweak the spacing to make it work out:

```suggestion
   -x
// ^ operator
//  ^ variable
```

Reply:
Ugh, you're totally right!  I'll fix that.